apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -402,7 +402,8 @@ const SwapForm = () => {
       const { account: toAccount, parentAccount: toParentAccount } = to;
       const fromAccountId = accountToWalletAPIAccount(fromAccount, fromParentAccount)?.id;
       const toAccountId = accountToWalletAPIAccount(toAccount, toParentAccount)?.id;
-      const fromAmount = transaction?.amount / Math.pow(10, from.currency.units[0].magnitude);
+      const fromAmount = +transaction?.amount;
```

Comment:
It would be better to explicit `Number(transaction?.amount)` or `parseInt` or `parseFloat`, the `+(...)` syntax isn't known by everyone and also is a bit too permissive for the typing / can lead to bugs. are we ok losing precision there btw?